frontend/src/components/TradeBox/index.js

```diff
@@ -72,6 +75,7 @@ class TradeBox extends Component {
       badInvoice: false,
       badAddress: false,
       badStatement: false,
+      turtle_mode: false,
```

Comment:
Out of curiosity, why not use camel case `turtleMode` here too? No fix needed as we are rewriting this component in parallel.

Reply:
My brain was out I guess, I can change it anyways to avoid confusion